facs/bloom.c

```diff
@@ -218,6 +218,7 @@ int save_bloom (char *filename, bloom * bl, char *prefix, char *target)
 {
   char *bloom_file = NULL;
   int fd;
+	int check_ftruncate = 1;
```

Comment:
It is a bit amusing given that coding style was mentioned earlier in the pull request, but fair enough, merging ;)